### ISSUE TITLE
Error while training the 1D CNN in keras with tensorflow backend 

### ISSUE THREAD

**Issue by ranjal** [tensorflow/tensorflow#25245]
Hello Everyone!!
I want to make a 1D CNN model using keras.
Consider that I've got a dataset X which is (2015930,13) where 13 is the number of features of audio where I want to do the 1d convolution on My Y is (2015930, 7) and I'm working with keras with tensorflow backend.

While training I am getting the following error. I am very new in this area. So I am unable to find the exact solution. The code of model is :
model = Sequential()
model.add(layers.Conv1D(40,2, activation='relu', input_shape=(13,1)))
model.add(layers.Conv1D(26,1, activation='relu'))
model.add(Flatten())
model.add(Dense(10, activation='relu'))
model.add(Dropout(0.2))
model.add(layers.Dense(y_train.shape[1],activation='softmax'))

UnknownError : Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
[[{{node conv1d_12/convolution/Conv2D}} = Conv2D[T=DT_FLOAT, _class=["loc:@train...propFilter"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="VALID", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](training_3/Adam/gradients/conv1d_12/convolution/Conv2D_grad/Conv2DBackpropFilter-0-TransposeNHWCToNCHW-LayoutOptimizer, conv1d_12/convolution/ExpandDims_1)]]
[[{{node loss_6/mul/_525}} = _Recvclient_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_770_loss_6/mul", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]]
My system specification are:
GPU system with NVIDIA graphics QuadroK2200.
OS: Ubuntu 16.04.1
print(tf.version) : 1.12.0(tensor flow)

Actually all the software were already installed by someone else so I am not sure CUDA and CUDNN are present or not.
So I used these command to check it.
cat /usr/local/cuda/version.txt : CUDA Version 9.0.176
cat /usr/local/cuda/include/cudnn.h | grep CUDNN_MAJOR -A 2 :
#define CUDNN_MAJOR 7
#define CUDNN_MINOR 0
#define CUDNN_PATCHLEVEL 5

#define CUDNN_VERSION (CUDNN_MAJOR * 1000 + CUDNN_MINOR * 100 +CUDNN_PATCHLEVEL)
#include "driver_types.h"

Can anyone help me why this error is coming ?
Please reply I am struggling since one week.
If any additional information require I can share it.

Thanks in advance

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
